Detectors/TPC/base/include/TPCBase/Defs.h

```diff
@@ -51,26 +51,47 @@ enum class PadSubset : char {
   Partition,  ///< Partitions (up to 36*5)
   Region      ///< Regions (up to 36*10)
 };
-/// Pad centres as 2D float
-typedef Point2D<float> PadCentre;
-typedef Point2D<float> GlobalPosition2D;
-typedef Point2D<float> LocalPosition2D;
-typedef Point3D<float> GlobalPosition3D;
-typedef Point3D<float> LocalPosition3D;
+/// default point definitions
+template <typename T>
+using Point3D     = ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::DefaultCoordinateSystemTag>;
+template <typename T>
+using Point3Dglob = ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::GlobalCoordinateSystemTag>;
+template <typename T>
+using Point3Dloc  = ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::LocalCoordinateSystemTag>;
```

Comment:
Ok, I see. I can also again remove the local and global part and move to default. I only thought it would be nice to have this distinction.
I'll remove it again then until there are further discussions.